netbout-web/src/test/java/com/netbout/dynamo/DyMessagesITCase.java

```diff
@@ -153,4 +159,31 @@ public void codeMarkdownInMessages() throws Exception {
             Matchers.equalTo("    4 leading spaces retained")
         );
     }
+    
+    /**
+     * DyMessages has to throw {@code HttpException} if invoke {@code unread()}
+     * for user which is not in a bout.
+     * @throws URISyntaxException If there is some problem inside
+     * @throws IOException If there is some problem inside
+     */
+    @Test(expected = HttpException.class)
+    public void exceptionIfBoutNotFound() throws IOException {
```

Comment:
@original-brownbear 
1th is not realy true: i don't catch messages.unread() because i want to see exception. 
2nd i see just 

> It's a good practice to always declare test methods as throwing Exception.

but i really don't see any explanation. However it doesn't matter here, i will change it.